app/(app)/feature-flag-example/_client.tsx

```diff
@@ -1,26 +0,0 @@
-"use client";
```

Comment:
This whole folder should have been left where it was an untouched. It is just to use as an example to show how to do feature flags.